File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
import java.util.*;
class Solution {
    public int[] solution(String[] wallpaper) {
        int minX = 100;
        int minY = 100;
        int maxX = 0;
        int maxY = 0;
        for(int i = 0; i < wallpaper.length; i++) {
            for(int j = 0; j < wallpaper[i].length(); j++) {
                char c = wallpaper[i].charAt(j);
                if (c == '#') {
                    if(minX > j) minX = j;
                    if(maxX < j) maxX = j;
                    if(minY > i) minY = i;
                    if(maxY < i) maxY = i;
                    
                }
            }
        }
        System.out.print(minX + " " + maxX + " " + minY + " " + maxY);
        int[] answer = {minY, minX , maxY+1, maxX+1};
        
        
        
        return answer;
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    var answer = [];
    let minY = Infinity;
    let maxY = -Infinity;
    let minX = Infinity;
    let maxX = -Infinity;
    
    
    wallpaper.forEach((v,i) => {
        if(v.includes('#')) {
            minY = Math.min(minY,i);
            maxY = Math.max(maxY,i);
        }
    })
    for (let i = 0; i < wallpaper.length; i++) {
        for (let j = 0; j < wallpaper[i].length; j++) {
            if(wallpaper[i][j] === '#') {
                minX = Math.min(minX,j);
                maxX = Math.max(maxX,j);
            }
        }
    }
    answer = [minY,minX,maxY+1,maxX+1];
    return answer;
}